client/main.lua

```diff
@@ -107,7 +107,7 @@ local function askToLeave()
 	if JailTime > 0 then
 		exports.qbx_core:Notify( Lang:t("info.timeleft", {JAILTIME = JailTime}))
 	else
-		release()
+		TriggerServerEvent("qbx_prison:server:releasePlayer")
```

Comment:
Rename event to something like "qbx_prison:server:prisonTimeExpired"
Event names should be about in what situation it's being triggered, not what handlers should exist for it.
https://manason.github.io/effective-fivem-lua/events/#past-tense